index.html

```diff
@@ -3325,6 +3323,48 @@ <h2>
       <h2>
         Security Considerations
       </h2>
+      <section class="informative">
+        <h2 id="sec-enhancements">
+          Mechanisms to enhance security
```

Comment:
Note that this deals with both privacy and security aspects. I think we should just fold this into the parent section.

Reply:
+1 - also "enhancements" sounds strange to my ear, but then I'm accustomed to thinking in terms of "threats" and "mitigations".